doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -913,6 +923,20 @@ Note that if a custom payload is used for the notification (the field `payload`
 corresponding subscription), then a value of `custom` is used for the `Ngsiv2-AttrsFormat` header
 in the notification.
 
+## Special metadata in notifications
+
+The following metadata are not included in notifications by default, the `metadata` field (within
+`notification` field) in subscription is used to do so.
+
+* `previousValue`: the value and type of this metadata is the one of the attribute to which it
+  is attached previous to the processing of the request that triggered the notification. If the
+  type/value of `previousValue` is the same type/value than in the attribute in which it has been
+  included, then the attribute has not actually changed its value.
+
+* `actionType`: it is included if the attribute to which it is attached was included in the request
+  that triggered the notification. Its value depend on the request operation type: `update`
+  for updates, `append` for creation and `delete` for deletion. Its type is always `Text`.
```

Comment:
It would be interesting, future wise, to distinguish between update and change action. so my suggestion here is to use now `change' or 'valueChange' to identify the changes currently being notified by the Orion implementation.